platforms/platforms.go

```diff
@@ -220,6 +220,9 @@ func Parse(specifier string) (specs.Platform, error) {
 		// we have a fully specified variant, this is rare
 		p.OS = normalizeOS(parts[0])
 		p.Architecture, p.Variant = normalizeArch(parts[1], parts[2])
+		if p.Architecture == "arm64" && p.Variant == "" {
+			p.Variant = "v8"
```

Comment:
So, this sets it to "v8" if its present?

Reply:
This sets it to "v8" if it was given as "v8" (or "8"). This `Parse` function is kind of confusing in that it normalizes the parts but not the whole. Take a look at the test cases I added to make sure it makes sense.

Reply:
Diff of both changes in the release backport PR https://github.com/containerd/containerd/pull/2416/files